.markdownlint.yaml

```
@@ -4,6 +4,11 @@ default: true
 # hard-tabs
 MD010: false
 
+# 80 char line-length
+MD013:
+  code_blocks: false
```

```suggestion
  ignore_code_blocks: true
```

like in the other PR

---

It looks like `code_blocks` is the correct parameter for the version of markdownlint that we use:

```
make
./goyek.sh
===== TASK  markdownlint
Cmd: docker run --rm -v /home/tyler/go/src/github.com/signalfx/splunk-otel-go:/markdown 06kellyjac/markdownlint-cli:0.28.1 **/*.md
docs/troubleshooting.md:30:81 MD013/line-length Line length [Expected: 80; Actual: 84]
docs/troubleshooting.md:39:81 MD013/line-length Line length [Expected: 80; Actual: 164]
exit status 1
----- FAIL: markdownlint (1.09s)
task failed	3.771s
exit status 1
make: *** [Makefile:37: goyek] Error 1
make  7.82s user 3.10s system 269% cpu 4.055 total
```

---

you are correct we would have to bump markdownlint